modules/utxo-lib/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bitgo/utxo-lib",
-  "version": "2.1.0",
+  "version": "2.3.0",
```

Comment:
This version bump is just for the dependency change? I notice this skips over the most recently published 2.2.0x version.

Reply:
I was getting strange stale dependency errors prior to this change